### ISSUE TITLE
Default repositories: must be able to specify multiple repos

### ISSUE THREAD

**Issue by dikim33** [oscar-cluster/oscar#574]
**Reported by valleegr on 27 Oct 2009 22:32 UTC**
At least on ubuntu, we need to be able to specify multiple default repo for the distro. For instance, to package Ganglia, we need the two following entries:
```
$ more /tftpboot/distro/ubuntu-904-x86_64.url
# Generated file; please do not manually edit

http://us.archive.ubuntu.com/ubuntu/+jaunty+main+universe+multiverse
http://us.archive.ubuntu.com/ubuntu/+jaunty-updates+main+universe+multiverse
```

**Comment by dikim33** [comment]
**Comment by valleegr on 29 Oct 2009 15:33 UTC**
For centos 5.3, we need to include http://mirror.centos.org/centos/5.3/extras/x86_64/

**Comment by dikim33** [comment]
**Comment by valleegr on 29 Oct 2009 21:33 UTC**
(In [8933]) This check-in fixes #574 - We can now specify multiple default distro repositories. For that,
we changed the syntax of the "supported_distros.txt" configuration file.
